pkg/secrets/plugins/in-memory/store.go

```diff
@@ -36,36 +28,13 @@ func NewStore() *Store {
 	return s
 }
 
-func (s *Store) Connect() error {
```

Comment:
I realized that a lot of this code was unused and leftover from when it was repurposed from the older storage protocol.